api/serverpb/server.proto

```diff
@@ -83,7 +83,7 @@ message CheckUpdatesResponse {
   // True if pmm-update can be used for an update.
   bool pmm_update_available = 6;
```

Comment:
I would rename this field also to make it more clear.
maybe make them like `internal_pmm_update_available` and `external_pmm_update_available`?

Reply:
`pmm_update_service_available`?
The word `service` implies to me it's not the act of an update, rather a service. At least for me this is helpful. Looking for feedback here though.

Reply:
`pmm_update_available` sounds like a new update(version) for PMM is available which might be not correct.